hiring-process.md

```diff
@@ -2,7 +2,7 @@
 
 We strive to be intentional in our hiring process. We believe very strongly in the role of a responsible, local business to fully represent the demographics of their community. This takes time and effort on our part to grow a more diverse network and we are actively taking measures to ensure that this is the case.
```

Comment:
"local" business here might want to be reworded, since so many of us live outside of the area that was described here originally.